File: tcpclient.java
import java.io.*;
import java.net.*;
import java.util.Scanner;

public class tcpclient {
    public static void main(String[] args) {
        String address = "localhost";
        int host = 5000;

        try (Socket socket = new Socket(address, host)) {
            System.out.println("Connected to server at " + address + " on port " + host);

            BufferedReader in = new BufferedReader(new InputStreamReader(socket.getInputStream()));
            PrintWriter out = new PrintWriter(socket.getOutputStream(), true);
            Scanner scanner = new Scanner(System.in);

            String userInput;
            while (true) {
                System.out.print("Enter message (type 'exit' to close the connection): ");
                userInput = scanner.nextLine();
                out.println(userInput);
                System.out.println(userInput);
                if (userInput.equalsIgnoreCase("exit")) break;
                
                String responseFromServer = in.readLine();
            }
            scanner.close();
        } catch (IOException e) {
            e.printStackTrace();
        }
    }
}

File: tcpclient.py
import socket

HOST = '127.0.0.1'
PORT = 65432

with socket.socket(socket.AF_INET, socket.SOCK_STREAM) as client_socket:
    client_socket.connect((HOST, PORT))
    while True:
        message = input("Enter message to send (type 'exit' to quit): ")
        if message.lower() == 'exit':
            break
        client_socket.sendall(message.encode())
        data = client_socket.recv(1024)
        print(f"Received from server: {data.decode()}")